magento/pwa-studio
# [PWA-628] My Account: Wishlist: Main View

## Description (from JIRA ticket)

As a shopper, when logged in and viewing the favorites/wishlist page I should see specific details about any created lists. 

In-Scope: 
Build main view of favorites/wish list page for both desktop and mobile 

Requirements
- Ensure empty state (no lists exist) view is created 
- build view for Wishlists w/ all Icons/CTAs included in Design (create/remove/move/kebab menu/add to cart) 
- workflows for each CTA are delivered in follow on stories 
- build view for an item included in the a list 
- CE: only supports creation of 1 list 
- EE: allows for multiple lists 

## Related Issue





* [[PWA-628](https://jira.corp.magento.com/browse/PWA-628)] My Account: Wishlist: Main View

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


Data is mocked right now, so only a few test cases to check (this requires you're using a backend with Venia Sample Data).
1. Login as a Customer
2. Using either desktop account menu or left nav, open Favorites Lists
3. Verify one list is rendered with two products (one simple, one config). Look at both desktop, tablet, and mobile views.
4. Try adding both items to your cart, verify counter updates when done and product is in cart

## Screenshots / Screen Captures (if appropriate)
Example of an error, which would be hard to trigger w/o backend access (Toast)
![](https://i.gyazo.com/0dcad47d8cd8e443cf7940469910a2db.gif)

## Checklist

* I have updated the documentation accordingly, if necessary.
